lib/client/fileUpload.js

```diff
@@ -47,11 +51,18 @@ Template.afFileUpload.helpers({
   },
   uploadedFile() {
     const template = Template.instance();
-    const _id = template.fileId.get() || this.value;
-    if (typeof _id !== 'string' || _id.length === 0) {
-      return null;
-    }
-    return template.collection.findOne({_id:_id});
+    var _id = template.fileId.get() || this.value;
+    if (typeof _id !== 'string' || _id.length === 0) return null;
+    const file = template.collection.findOne({_id:_id});
+    if (!file) return null;
+    file.hasBootstrap = template.hasBootstrap();
```

Comment:
We don't need it if `hasBootstrap` is the template helper

Reply:
Then `hasBootstrap` is not available in in the image preview template anymore

Reply:
has been fixed in the latest commit